### PR TITLE
Packages/did: Add `keyType` Parameter to `createDid` Method

### PR DESCRIPTION
**Pull request by belyrs**

### CODE REVIEW

File: packages/did/src/Did.chain.ts [hunk 1]
```diff
@@ -515,21 +516,23 @@ export async function fetchFromMnemonic(mnemonic: string): Promise<DidDocument>
  */
 export async function createDid(
   submitterAccount: CordKeyringPair,
-  theMnemonic?: string,
+  keytype?: string,
```
Comment (vatsa287) [inline packages/did/src/Did.chain.ts]: Can we do this `keytype: string = 'sr25519'`

File: packages/did/src/Did.chain.ts [hunk 2]
```diff
@@ -515,21 +516,23 @@ export async function fetchFromMnemonic(mnemonic: string): Promise<DidDocument>
  */
```
Comment (vatsa287) [inline packages/did/src/Did.chain.ts:516]: Updating the method docs would be good to have.
Mention `keytype`, `mnemonic` is optional.

**Review by vatsa287 — COMMENTED** [review]
Some minor comments.

**Review by vatsa287 — COMMENTED** [review]
LGTM

**Review by amarts — APPROVED** [review]

**Pull request merged**